### ISSUE TITLE
Probable false positive when linting action.yml on the image directive

### ISSUE THREAD

**Issue by ferrarimarco** [rhysd/actionlint#428]
Hi, and thanks for working on actionlint! :)

After updating to 1.7.0, actionlint reports what it's likely a false positive.

As part of the test suite of a container-based action, we do the following:

1. Build the container image. The resulting image is tagged as `ghcr.io/super-linter/super-linter:latest`
2. Update the `image` section of `action.yml` to point to the container image we just built. The resulting `action.yml` is similar to the following:

```yml
name: 'Super-Linter'
author: 'Super-linter contributors'
description: 'Super-linter is a ready-to-run collection of linters and code analyzers, to help validate your source code.'
runs:
  using: 'docker'
  image: 'ghcr.io/super-linter/super-linter:latest'
```

3. Run the action using the container image we just built:

```yml
      - name: Test Local Action (debug log)
        uses: ./
        env:
          ...
```

As part of this test suite, we also run actionlint. It gets confused when looking at the updated `action.yml` as follows:

```text
.github/workflows/ci.yml:163:15: file "ghcr.io/super-linter/super-linter:latest\\" does not exist in "/github/workspace". it is specified at "image" key in "runs" section in "Super-Linter\\" action [action]
      |
  163 |         uses: ./
      |               ^~
  .github/workflows/ci.yml:163:15: the local file "ghcr.io/super-linter/super-linter:latest" referenced from "image" key must be named "Dockerfile" in "Super-Linter" action. the action is defined at "/github/workspace" [action]
      |
  163 |         uses: ./
      |               ^~
```

**Comment by ferrarimarco** [comment]
Or, maybe we should add `docker://` I suppose?

**Comment by rhysd** [comment]
Thank you for catching this. I think actionlint is too strict on checking the `image` property.

> Or, maybe we should add docker:// I suppose?

It should fix the error. But if `ghcr.io/...` is valid on GitHub Actions, actionlint should not cause an error on the value.

**Comment by rhysd** [comment]
I searched GitHub: https://github.com/search?q=%22using%3A+%27docker%27%22+language%3AYAML+ghcr.io&type=code&p=1

- In most cases, `docker://` is explicitly specified. I recommend this because removing the scheme is ambiguous
- actionlint should not complain the missing scheme because it is a wrong usage. I'll fix this.
